src/main/java/com/google/devtools/build/lib/exec/SymlinkTreeHelper.java

```diff
@@ -85,16 +87,24 @@ public Path getOutputManifest() {
    * @throws CommandException
    */
   public void createSymlinksUsingCommand(
-      Path execRoot, BinTools binTools, ImmutableMap<String, String> shellEnvironment)
+      Path execRoot,
+      BinTools binTools,
+      ImmutableMap<String, String> shellEnvironment,
+      OutputStream outStream,
```

Comment:
I'd prefer to pass in an OutErr object, and then check for it's null-ness below.